Dockerfile

```diff
@@ -83,42 +92,69 @@ RUN curl -L --output /tmp/minc-toolkit-1.9.18.deb https://packages.bic.mni.mcgil
 #Enable minc-toolkit
 RUN echo '. /opt/minc/1.9.18/minc-toolkit-config.sh' > /etc/profile.d/98minc.sh
 
+# minc-toolkit configuration parameters for 1.9.18-20200813
+ENV MINC_TOOLKIT=/opt/minc/1.9.18 \
+  MINC_TOOLKIT_VERSION="1.9.18-20200813"
+ENV PATH=${MINC_TOOLKIT}/bin:${MINC_TOOLKIT}/pipeline:${PATH} \
+  PERL5LIB=${MINC_TOOLKIT}/perl:${MINC_TOOLKIT}/pipeline${PERL5LIB:+:$PERL5LIB} \
+  LD_LIBRARY_PATH=${MINC_TOOLKIT}/lib:${MINC_TOOLKIT}/lib/InsightToolkit${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH} \
+  MNI_DATAPATH=${MINC_TOOLKIT}/../share:${MINC_TOOLKIT}/share \
+  MINC_FORCE_V2=1 \
+  MINC_COMPRESS=4 \
+  VOLUME_CACHE_THRESHOLD=-1 \
+  MANPATH=${MINC_TOOLKIT}/man${MANPATH:+:$MANPATH} \
+  # integrated ANTs tools
+  ANTSPATH=${MINC_TOOLKIT}/bin
+
 #Enable ANTs
 RUN echo 'export PATH=/opt/quarantine/ANTs/bin${PATH:+:$PATH}' > /etc/profile.d/99ANTS.sh
 RUN echo 'export ANTSPATH=/opt/quarantine/ANTs/bin' >> /etc/profile.d/99ANTS.sh
+ENV PATH=/opt/quarantine/ANTs/bin${PATH:+:$PATH} \
+  ANTSPATH=/opt/quarantine/ANTs/bin
+ENV PATH=/opt/ANTs/bin${PATH:+:$PATH}
+RUN echo 'export PATH=/opt/ANTs/bin${PATH:+:$PATH}' >> /etc/profile.d/99ANTs.sh
+
+USER rabies
+WORKDIR /home/rabies
+ENV HOME="/home/rabies"
 
 #install conda
 RUN curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
-    bash Miniforge3-Linux-x86_64.sh -b -p /opt/quarantine/miniforge && \
+    bash Miniforge3-Linux-x86_64.sh -b -p ${HOME}/miniforge && \
     rm -f Miniforge3-Linux-x86_64.sh
+ENV CONDA_DIR=${HOME}/miniforge
 
 #enable conda
-RUN cp /opt/quarantine/miniforge/etc/profile.d/conda.sh /etc/profile.d/99conda.sh
-RUN echo 'conda activate' >> /etc/profile.d/99conda.sh
-RUN echo 'export PATH=/opt/ANTs/bin${PATH:+:$PATH}' >> /etc/profile.d/99ANTs.sh
+RUN echo '. ${HOME}/miniforge/etc/profile.d/conda.sh' >> ${HOME}/.bashrc
+RUN echo 'conda activate' >> ${HOME}/.bashrc
 
-COPY rabies /src/rabies
-COPY minc-toolkit-extras /src/minc-toolkit-extras
-COPY twolevel_ants_dbm /src/twolevel_ants_dbm
-COPY scripts /src/scripts
-COPY setup.py MANIFEST.in README.md LICENSE dependencies.txt /src/
+ENV CONDA_EXE='${CONDA_DIR}/bin/conda' \
+  CONDA_PYTHON_EXE='${CONDA_DIR}/bin/python' \
+  # override the path with the conda environment so that it becomes the default
+  PATH=${CONDA_DIR}/bin:$PATH
 
-RUN . /etc/profile.d/99conda.sh && conda config --append channels simpleitk && \
-  conda install -y 'networkx>=2.4' 'matplotlib>=3.1.1' 'nibabel>=2.3.1' 'nilearn>=0.4.2' 'nipype>=1.1.4' 'numpy>=1.16.2' 'pandas' 'scikit-learn>=0.20.0' 'scipy' 'simpleitk>=1.2.2' 'tqdm' 'pathos' && \
+ENV RABIES=${HOME}/RABIES
+RUN mkdir $RABIES
+
+COPY rabies $RABIES/rabies
```

Comment:
too many instances of the word "rabies" here in different variants

Reply:
Seems clear to me that one stands for the PATH and the other for the scripts. Do you have an alternative suggestion?